map/bookmark_manager.cpp

```diff
@@ -359,6 +359,24 @@ Track const * BookmarkManager::GetTrack(kml::TrackId trackId) const
   return (it != m_tracks.end()) ? it->second.get() : nullptr;
 }
 
+Track * BookmarkManager::GetTrackForEdit(kml::TrackId trackId)
+{
+    CHECK_THREAD_CHECKER(m_threadChecker, ());
+    auto it = m_tracks.find(trackId);
+    if (it == m_tracks.end())
+        return nullptr;
+    
+    return it->second.get();
+}
+
+
+void BookmarkManager::MoveTrack(kml::TrackId trackID, kml::MarkGroupId curGroupID, kml::MarkGroupId newGroupID)
+{
+    CHECK_THREAD_CHECKER(m_threadChecker, ());
```

Comment:
2 spaces instead of 4 here and in other places, please.

Reply:
I've changed where I've found